docs/workflows/create_sync_publish.rst

```diff
@@ -84,7 +84,8 @@ Create a Publication
         }
     ]
 
-``$ export PUBLICATION_HREF=$(http :24817/pulp/api/v3/publications/rpm/rpm/ | jq -r '.results[] | ._href')``
+``$ export PUBLICATION_HREF=$(http :24817/pulp/api/v3/publications/rpm/rpm/ | jq -r '.results[] | select(.repository_version|test("'$REPO_HREF'.")) | ._href') ``
+
```

Comment:
@goosemania i guess `test` is typo here.

Reply:
i think space before `` is the root cause of the rendering

Reply:
@ipanova , `test` is a way to use regexp in `jq`.
Thanks, I'll update to remove the space.